### ISSUE TITLE
Fatal error: Uncaught ArgumentCountError: Too few arguments to function

### ISSUE THREAD

**Issue by supermodo** [wp-graphql/wp-graphql-meta-query#13]
Hi, thank you for the very good plugins, they are really useful.

Anyway, at the moment I get this error (on mywpsite.com/graphql with wp in debug mode)

```
Fatal error: Uncaught ArgumentCountError: Too few arguments to function WPGraphQL\MetaQuery::add_input_fields(), 3 passed in /home/mywpsite/public_html/admin/wp-includes/class-wp-hook.php on line 288 and exactly 4 expected in /home/mywpsite/public_html/admin/wp-content/plugins/wp-graphql-meta-query-develop/wp-graphql-meta-query.php:139
Stack trace:
#0 /home/mywpsite/public_html/admin/wp-includes/class-wp-hook.php(288): WPGraphQL\MetaQuery->add_input_fields(Array, 'UserToAwardConn...', Array)
#1 /home/mywpsite/public_html/admin/wp-includes/plugin.php(206): WP_Hook->apply_filters(Array, Array)
#2 /home/mywpsite/public_html/admin/wp-content/plugins/wp-graphql-develop/src/Type/WPInputObjectType.php(40): apply_filters('graphql_input_f...', Array, 'UserToAwardConn...', Array)
#3 /home/mywpsite/public_html/admin/wp-content/plugins/wp-graphql-develop/src/Registry/TypeRegistry.php(440): WPGraphQL\Type\WPInputObjectType::prepare_fields(Array, 'UserToAwardConn...')
#4 [internal function]: WPGraphQL\Registry\TypeRegistry->WPGraphQL\Registr in /home/mywpsite/public_html/admin/wp-content/plugins/wp-graphql-meta-query-develop/wp-graphql-meta-query.php on line 139
```
and look like is referring to this php function in the plugin
```
public function add_input_fields( $fields, $type_name, $config, $type_registry ) {
    if ( isset( $config['queryClass'] ) && 'WP_Query' === $config['queryClass'] ) {
        $this->register_types( $type_name, $type_registry );
        $fields['metaQuery'] = [
            'type' => $type_name . 'MetaQuery',
        ];
    }
    return $fields;
}
```
I've disabled all plugins and activated only `wp-graphql` and `wp-graphql-meta-query` but still
the error is breaking the `wp-graphql` functionalities. I've almost the same configuration in another site and is working fine.
What I'm doing wrong?

**Comment by tonk379** [comment]
I am facing the same issue. 

I am getting a following warning after installing the plugin
**Warning:  Missing argument 4 for WPGraphQL\MetaQuery::add_input_fields()**
metaQuery does not seem to be working.
I am using PHP version 7.0.33
Where exactly the issue. i am getting the following error when i run the graphql query with metaQuery
**Field \"metaQuery\" is not defined by type RootQueryToPostConnectionWhereArgs; Did you mean dateQuery?** 
Please help.

**Comment by supermodo** [comment]
@tonk379 I didn't had time to dig into the problem yet but maybe this can help you.

I have 2 installations very similar (should be identical but look like their are not) in 2 different folders/subdomains (dev and production) in the same server with PHP 7.3.
Dev is working fine, prod is giving the error.

I've checked the ACF and wordpress settings: they look the same. Also htaccess is the same.
The only difference I've found yet is the amount of content.
